cogs/playerstats.py

```diff
@@ -20,10 +20,14 @@ def __init__(self, bot):
         self.bot = bot
 
         self.CLUB_ID_TRANSLATIONS = {
-            'arsenal': '18bb7c10',
-            'chelsea': 'cff3d9bb',
-            'spurs': '361ca564'
-        }
+            # 'team': ['club_id', 'icon_url']
+            'arsenal': ['18bb7c10', 'https://resources.premierleague.com/premierleague/badges/t3.png'],
+            'chelsea': ['cff3d9bb', 'https://resources.premierleague.com/premierleague/badges/t8.png'],
+            'spurs': ['361ca564', 'https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/120/apple/325/pile-of-poo_1f4a9.png'],
+            'united': ['0bbd83f6', 'https://resources.premierleague.com/premierleague/badges/t1.png'],
+            'brentford': ['cd051869', 'https://resources.premierleague.com/premierleague/badges/t94.png'],
+            'liverpool' : ['822bd0ba', 'https://resources.premierleague.com/premierleague/badges/t14.png']
+            }
```

Comment:
It looks like the only thing that changes between teams it the image. Could this be changed to reflect that fact?